webbpsf/roman.py

```diff
@@ -1,6 +1,6 @@
 """
 ==============================
-WFIRST Instruments (pre-alpha)
+Roman Instruments (pre-alpha)
```

Comment:
We should delete the comment "(pre-alpha)" here, which is no longer true (and hasn't been for a long while! I hadn't realized this was still stated like that...)